account_invoice_payment_term_date_due/views/account_move.xml

```diff
@@ -5,23 +5,23 @@
     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 -->
 <odoo>
-
     <record id="view_move_form" model="ir.ui.view">
         <field name="model">account.move</field>
         <field name="inherit_id" ref="account.view_move_form" />
         <field name="arch" type="xml">
-            <field name="invoice_payment_term_id" position="after">
-                <span
-                    name="invoice_payment_term_date_due"
-                    class="text-muted oe_read_only"
+            <xpath expr="//field[@name='invoice_payment_term_id']" position="after">
+                <i
+                    class="fa fa-arrow-right mx-3 text-muted"
                     attrs="{'invisible': ['|', ('invoice_payment_term_id', '=', False), ('invoice_date_due', '=', False)]}"
-                    t-translation="off"
-                >
-                    <i class="fa fa-arrow-right mx-3" />
-                    <field name="invoice_date_due" />
-                </span>
-            </field>
+                    style="padding-left: 30px;padding-top: 2px;"
+                />
+                <field
+                    class="text-muted"
+                    name="invoice_date_due"
+                    force_save="1"
+                    attrs="{'readonly': [('invoice_payment_term_id', '!=', False)], 'invisible': ['|', ('invoice_payment_term_id', '=', False), ('invoice_date_due', '=', False)]}"
+                />
+            </xpath>
```

Comment:
I found it easier to read before, with the `invisible` attribute being set on the parent span directly. 🤔 Same for the `text-muted` class..
Is there a reason for this change?

Reply:
What I mean is that you can keep the `span` while achieving that, which also results in a minimal diff
For example, something like this:

```xml
            <field name="invoice_payment_term_id" position="after">
                <span
                    name="invoice_payment_term_date_due"
                    class="text-muted"
                    attrs="{'invisible': ['|', ('invoice_payment_term_id', '=', False), ('invoice_date_due', '=', False)]}"
                    t-translation="off"
                >
                    <i class="fa fa-arrow-right mx-3" />
                    <field
                        name="invoice_date_due"
                        force_save="1"
                        attrs="{'readonly': [('invoice_payment_term_id', '!=', False)]}"
                    />
                </span>
            </field>
```

Reply:
@ivantodorovich I tried to do it like you suggest:

![view](https://github.com/OCA/account-invoicing/assets/85480719/8c6193c6-25a4-4efc-b2f1-fee19c05ebe3)


And in non edit mode it's okay:

![image](https://github.com/OCA/account-invoicing/assets/85480719/76a33888-bba0-4a8f-b365-cd9acf8a74a7)

But in edit mode it is not displayed correctly:

![image](https://github.com/OCA/account-invoicing/assets/85480719/e19a6c0d-c91b-4316-9d67-29426ff78665)